FtpClient/FtpCli/InitializeSession.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+
+namespace FtpCli
+{
+  // The IntializeSession class prompts the user for session details
+  // which are stored as a Data object
+  public class InitializeSession
```

Comment:
we should probably have an option to initialize with variables immediately. If they send in the username and server through the command line on start, we don't want to re-prompt them. Or are we not accepting any command line arguments when running the application?

Reply:
Sounds reasonable. Added changes to allow for command line variables with `dotnet run server user` using the existing validation functions called through the `initialPrompt` method. Decided to do this instead of a parameterized constructor to avoid copying the cli args unnecessarily.